src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -0,0 +1,803 @@
+      rowClass = rowChild
+        ? DATA_TABLE_CLASSES.ROW_CHILD
+        : DATA_TABLE_CLASSES.ROW;
+
+    if (this._expandable) {
+      if (bodyRow.accordion) {
+        rowCells.push(this.expansionButton(rowId));
+        rowAccordionContent.push(this.rowAccordionContent(bodyRow.accordion));
+      } else {
+        rowCells.push(
+          <div
+            class={`${DATA_TABLE_CLASSES.CELL} ${DATA_TABLE_CLASSES.EXPANDABLE}`}
+            role="cell"
+          />
+        );
+      }
+    }
+
+    if (this.data.config.selectable) {
+      rowCells.push(this.selectRowCheckbox(rowId, false));
+    }
+
+    let cellIndex = 0;
+
+    bodyRow.data.map((rowCell: any) => {
+      const columnSettings = this.data.head[
+          Object.keys(this.data.head)[cellIndex]
+        ],
+        alignment = this.cellAlignment(
+          rowCell.align ? rowCell.align : columnSettings.align || null
+        ),
+        cellLabel = rowCell.label || columnSettings.label,
+        cellWidth =
+          this.data.config.columnSizes && this.currentScreenBreakpoint
+            ? this.data.config.columnSizes[this.currentScreenBreakpoint][
+                cellIndex
+              ]
+            : null;
+      let cellData;
+
+      if (rowCell.template && this.$scopedSlots[rowCell.template]) {
+        if (rowCell.payload) {
+          cellData = this.$scopedSlots[rowCell.template]!(rowCell.payload);
+        }
+      } else if (typeof rowCell === 'string') {
+        cellData = (
+          <Tooltip message={rowCell} class="-w--100">
+            <div class={UTILITY_CLASSES.TYPOGRAPHY.TEXT_TRUNCATE}>
+              {rowCell}
+            </div>
+          </Tooltip>
+        );
+      } else {
+        cellData = (
+          <Tooltip message={rowCell.value} class="-w--100">
+            <div class={UTILITY_CLASSES.TYPOGRAPHY.TEXT_TRUNCATE}>
+              {rowCell.value}
+            </div>
+          </Tooltip>
+        );
+      }
+      rowCells.push(
+        <div
+          aria-label={cellLabel}
+          class={`
+            ${DATA_TABLE_CLASSES.CELL}
+            ${alignment}
+            ${cellWidth && cellWidth > 0 ? `-flex-basis--${cellWidth}` : ''}`}
+          data-label={cellLabel}
+          role="cell"
+          style={`
+            ${cellWidth === 0 ? 'display: none' : ''}
+            ${columnSettings.allowOverflow ? 'overflow: visible;' : ''}
+            `}
+        >
+          {cellData}
+        </div>
+      );
+
+      cellIndex++;
+    });
+
+    row.push(
+      <div
+        id={rowId}
+        class={`
+        ${rowClass}
+        ${this.selectedRows.includes(rowId) ? ACTIVE_CLASS : ''}
+        `}
+        role="row"
+      >
+        {rowCells}
+      </div>
+    );
+
+    if (bodyRow.accordion) {
+      row.push(
+        <transition name="slide-fade">
+          <div
+            v-show={this.accordionsShown.includes(rowId)}
+            id={`row-${rowId}-content`}
```

Comment:
@janibolkvadze, us a unique id for the data table in addition to the existing. That will make it possible to have more than one data-table per page.
```suggestion
            id={`${dataTableId}-row-${rowId}-content`}
```

Reply:
Solved.